### PR TITLE
Aug05

### PR DESCRIPTION
**Pull request by OlgaPa95**

### CODE REVIEW

**Review by ahsansiddiki — CHANGES_REQUESTED** [review]
Hi Olga, pls exclude node_modules folder from PR. You can add the folder into .gitignore

**Comment by OlgaPa95** [comment]
Thank you for your feedback, Ahsan. I noticed that after the push and was
wondering what is right to do in that case. I will redo that and make a
pull request after .

**Comment by OlgaPa95** [comment]
Hey Ahsan, I was trying to figure out how I can exclude the node_modules
folder from PR and I'm a little confused. Could you help me with that? Can
I do that in gitHub or should I exclude that from IDE, and then commit
again?

вт, 16 авг. 2022 г. в 19:44, Olga Paraschiv ***@***.***>:

File: test/specs/Class3/Homework.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        * 
+        * Verify user gets error message when submit empty login form
+        * expected error msg -> The email address or mobile number you entered isn't connected to an account.
+        * 
+        */
+    describe('Facebook.com tests', async()=>{
+        it('Verify user gets error message when submit empty login form', async ()=>{
+            await browser.url('https://www.facebook.com');
+            const loginButtonClick = await $('button[name=login]').click(); 
+            const errorMsg = await $('._9ay7').getText();
```
Comment (ahsansiddiki) [inline test/specs/Class3/Homework.js:61]: This class value looks like random, try to use a more reliable one.

File: test/specs/Class3/Homework.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        await browser.url('https://darksky.net/');
+        const searchBarInput = await  $('input[type=text]'); 
+        await searchBarInput.setValue('36542');
+        await browser.pause(3000);
+        await $('.searchButton').click();
+        await browser.pause(7000);
+
+        const pageTitle = await browser.getTitle();
+        const expectedPageTitle = "Dark Sky - Gulf Shores, AL"
+        expect(pageTitle, ' Wrong location displayed').to.equal(expectedPageTitle);
```
Comment (ahsansiddiki) [inline test/specs/Class3/Homework.js:43]: For this test it doesn't make sense to me to verify pageTitle. I would request to validate whether temp displayed or not?

**Review by ahsansiddiki — APPROVED** [review]

**Pull request merged**